overlord/snapshotstate/backend/helpers.go

```diff
@@ -152,26 +153,53 @@ func allUsers() ([]*user.User, error) {
 	return users, nil
 }
 
+var (
+	sysGeteuid   = sys.Geteuid
+	execLookPath = exec.LookPath
+)
+
+func pickUserWrapper() string {
+	// runuser and sudo happen to work the same way in this case.
+	// The main reason to prefer runuser over sudo is that runuser
+	// is part of util-linux, which is considered essential,
+	// whereas sudo is an addon which could be removed.
```

Comment:
Hold on, but the rationale for adding this branch is that we ... don't have runuser everywhere, right?

Reply:
May be worth mentioning in the comment that runuser is avaialble since util-linux 2.23 and some old systems that we support are on an even older version.